app/src/main/java/com/kuit/kuit6android/ui/cart/component/totalamount/DetailedTotalRow.kt

```
@@ -0,0 +1,66 @@
+                style = CoupangEatsTheme.typography.body_02_SB_12,
+            )
+        }
+        Row(
+            modifier = modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = "배달팁",
+                style = CoupangEatsTheme.typography.body_02_SB_12,
+            )
+            Text(
```

detekt (1.23.8)

[warning] String.format("%,d", totalAmount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)